integration_tests/mockApis/sendLegalMail.ts

```diff
@@ -282,14 +282,15 @@ const stubMoreChecksRequestedForBarcode = (): SuperAgentRequest =>
     },
   })
 
-const stubCreateBarcode = (): SuperAgentRequest =>
-  stubFor({
+const stubCreateBarcode = (prisonerName?: string): SuperAgentRequest => {
```

Comment:
We can create cypress tasks that setup wiremock stubs for specific people, or anyone if not specified 👍